modules/serverless-describe-broker-kn.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-using-brokers.adoc
+
+[id="serverless-describe-broker-kn_{context}"]
+= Describe an existing broker using the kn CLI
+
+.Prerequisites
+* The {ServerlessOperatorName}, Knative Serving and Knative Eventing are installed on your {product-title} cluster.
+* You have installed the `kn` CLI.
+
+.Procedure
+. Describe an existing broker:
++
+
+[source,terminal]
+----
+$ kn broker describe <broker_name>
+----
+
+// Add example output
```

Comment:
@rhuss same for this please :pray:

Reply:
Hi @abrennan89, this example output is missing, is it ok?